RecoLocalCalo/HGCalRecProducers/plugins/HGCalCLUEAlgo.cc

```diff
@@ -518,7 +518,10 @@ void HGCalCLUEAlgoT<T>::computeThreshold() {
 
   std::vector<double> dummy;
   const unsigned maxNumberOfThickIndices = 3;
-  dummy.resize(maxNumberOfThickIndices + 1, 0);  // +1 to accomodate for the Scintillators
+  if (isNose_)
```

Comment:
simplify to: `dummy.resize(maxNumberOfThickIndices + isNose_, 0);`

Reply:
done